corehq/apps/accounting/invoice_pdf.py

```diff
@@ -325,9 +325,14 @@ def draw_statement_period(self):
         origin_x = inches(0.5)
         origin_y = inches(7.15)
         self.canvas.translate(origin_x, origin_y)
+        if self.is_prepayment:
```

Comment:
not going to block on this, but curious why this string isn't translated. It seems like it should be, and that we should translate the entire string (e.g. "Prepayment term {} from {} to {}").  Happy to leave this to be resolved in a later PR.

Reply:
None of the text on the invoice template is translated. I think there might be a lower-priority goal to update that in the future, but my guess is that since the PDF template is very based on exact sizes (lines drawn in inches, etc) that there was concern about different language text of unknown length breaking the formatting.